index.html

```diff
@@ -2326,6 +2327,101 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains a remark expressing a reaction to other content within the page.</p>
+				<p>The relationship of comments to other content SHOULD be exposed:</p>
+				<ol>
+					<li>When the comment refers to other content within the page, the source content SHOULD use <pref>aria-details</pref> to point to the related comment.</li>
+					<li>When the comment is a reply to another comment:
+						<ul>
+							<li>If possible, the reply comment SHOULD be a semantic descendant of the comment it relates to, either as a DOM descendant element, or via <pref>aria-owns</pref>. In this case, user agents MUST automatically compute the level and expose it to assistive technologies. This is the preferred method for defining
```

Comment:
```suggestion
							<li>The reply comment SHOULD be a semantic descendant of the comment it relates to, either as a DOM descendant element, or via <pref>aria-owns</pref>. In this case, user agents MUST automatically compute the level and expose it to assistive technologies. This is the preferred method for defining
```

(I think the "If possible" just waters down the SHOULD. After all, if it's not possible, then authors will have to ignore the SHOULD).

Reply:
I just think it's clearer in the original, because even though that's technically correct, it lets me know that there can be times where it's not possible. Specifically, in the paginated comment use case. 

It could be changed to, "If all ancestors are available in the DOM, the reply comment SHOULD ..."

Reply:
I like your suggested text, with "If all ancestors are available in the DOM, ..."

Reply:
I changed to "If all ancestor comments are available in the DOM"

Reply:
Ah, already committed. :)
I'll mark this resolved.